docs/src/model_creation/network_analysis.md

```diff
@@ -8,7 +8,7 @@ section
 Note, currently API functions for network analysis and conservation law analysis
 do not work with constant species (currently only generated by SBMLToolkit).
 
-## Network representation of the Repressilator `ReactionSystem`
+## [Network representation of the Repressilator `ReactionSystem`](@id network_analysis_repressilator_representation)
```

Comment:
I added in the reference tags, in case we might need them it is nice to have them in place (per previous reasoning).